src/jit/Backend.cpp

```diff
@@ -1345,6 +1345,10 @@ void JITCompiler::compileFunction(JITFunction* jitFunc, bool isExternal)
                 m_context.appendTrapJump(ExecutionContext::GenericTrap, sljit_emit_jump(m_compiler, SLJIT_JUMP));
                 break;
             }
+            case ByteCode::NopOpcode: {
+                sljit_emit_op0(m_compiler, SLJIT_NOP);
+                break;
+            }
```

Comment:
Nit) what about enabling this code only in debug mode? Or it doesn't matter?

Reply:
I agree! The patch is not ready yet